lib/fetcher.js

```diff
@@ -105,6 +105,8 @@ class FetcherBase {
       this[_readPackageJson] = readPackageJsonFast
     }
 
+    this.replaceRegistryHost = opts.replaceRegistryHost !== false
```

Comment:
Is there other `!== false` stuff opts parsing in pacote? I'd rather start on the right foot here and insist that one pass in a boolean here.

Reply:
this option was changed to a string